### PR TITLE
SEO-174304 - Xamarin page with redirect

### PR DESCRIPTION
**Pull request by amyprudencekingstone**
Hi @MallikaRK ,
Worked on page with redirect in Xamarin docs. Kindly review

### CODE REVIEW

File: Xamarin/ListView/Getting-started.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ N> To know more about obtaining our components, refer to these links for [Mac](
 
 I> Starting with v16.2.0.x, if you reference Syncfusion assemblies from the trial setup or from the NuGet feed, you also have to include a license key in your projects. Please refer to [Syncfusion license key](https://help.syncfusion.com/common/essential-studio/licensing/license-key/) to know about registering Syncfusion license key in your Xamarin application to use our components.
```
Comment (MallikaRK) [inline Xamarin/ListView/Getting-started.md:71]: why you missed the link? are you checked?

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

**Review by Choza-rajan — APPROVED** [review]

File: Xamarin/ListView/Getting-started.md (hunk 1 above)
Comment (anandhan-rajagopal) [reply Xamarin/ListView/Getting-started.md]: Addressed the changes Commit link: https://github.com/syncfusion-content/xamarin-docs/pull/620/commits/703e32cb423a51444c803903be219828abdafcd0

**Comment by AarthiArjunan** [comment]
Created new PR for these Changes in the latest hotfix branch.
PR Link: https://github.com/syncfusion-content/xamarin-docs/pull/682/files

**Pull request closed without merging**